cogs5e/gametrack.py

```diff
@@ -466,11 +468,24 @@ async def customcounter(self, ctx, name=None, *, modifier=None):
         await character.commit(ctx)
 
         if new_value - counter.value:
-            out = f"{str(counter)}\n({abs(new_value - counter.value)} overflow)"
+            overflow = new_value - counter.value
+            changed = counter.value - old_value
+            out = f"{str(counter)}\n({abs(overflow)} overflow)"
+            # Calculations for Change (with overflow)
+            if not operator or operator == 'mod':
+                change = f' ({changed:+})'
+            elif operator == 'set':
+                change = f' ({changed:+})'
         else:
             out = str(counter)
-
-        result_embed.add_field(name=counter.name, value=out)
+            # Calculations for Change (without overflow)
+            if not operator or operator == 'mod':
+                change = f' ({modifier:+})'
+            elif operator == 'set':
+                delta = modifier - old_value
+                change = f' ({delta:+})'
+
+        result_embed.add_field(name=f'{counter.name}{change}', value=out)
```

Comment:
Personally, I'm more of a fan of having the delta display next to the value rather than in the title. (Consistent with `!g hp`)

Reply:
`!g hp` Is just a text send instead of an embed, but I can definitely change that

Reply:
oh! well... you get the idea 😛

Reply:
Like this?
![image](https://user-images.githubusercontent.com/16567386/93009671-bbcaaf00-f551-11ea-8f88-a18d8429708f.png)

Reply:
yup!